### ISSUE TITLE
Unable to see Map View with demo-angular

### ISSUE THREAD

**Issue by muhmadtabrez** [benedictkhoo/nativescript-google-places-sdk#4]
Hi,

   I have cloned the current version of source code in to my system and i am able to build demo-angular using command 'tns build android', After successful build i am able to run the demo in my device using command tns run --device <device id>.
  The issue is i am unable to see the maps view in my when i am trying to select location biased from drawer 

android version : 9
tns version : 6.0.2

following is my logs 

Webpack build done!
Project successfully prepared (android)
Installing on device 854142b...
Successfully installed on device with identifier '854142b'.
Successfully transferred 0.js on device 854142b.
Successfully transferred 1.js on device 854142b.
Successfully transferred bundle.js on device 854142b.
Successfully transferred runtime.js on device 854142b.
Successfully transferred vendor.js on device 854142b.
Restarting application on device 854142b...
JS: HMR: Hot Module Replacement Enabled. Waiting for signal.
JS: Angular is running in the development mode. Call enableProdMode() to enable the production mode.
Successfully synced application org.nativescript.demoangular on device 854142b.
JS: Dynamically loading class com.google.android.libraries.places.api.model.zzp was unsuccessful. Will attempt to load class from alternative ClassLoader.
JS: Dynamically loading class com.google.android.libraries.places.api.model.zzw was unsuccessful. Will attempt to load class from alternative ClassLoader.

*****************************

**Comment by benedictkhoo** [comment]
Hi @muhmadtabrez , did you add the necessary API keys in `environment.ts`? 🤔

**Comment by muhmadtabrez** [comment]
@benedictkhoo ,

   Yes i did that i am suspecting that nativescript-google-maps-sdk version is not upgraded or you can suggest whats wrong.

**Comment by benedictkhoo** [comment]
@muhmadtabrez sorry to hear that. To be honest I'm not sure what happened. Let me see whether I can reproduce it on my end 🙇

**Comment by muhmadtabrez** [comment]
@benedictkhoo, 

    Please find the image for your reference from my device when i tap on drawer with location biased
![Screenshot_2019-11-21-12-39-25-977_org nativescript demoangular](https://user-images.githubusercontent.com/5879103/69316270-7786bf00-0c5e-11ea-8760-d0714ae82423.png)
